middle.mkd

```diff
@@ -274,8 +274,9 @@ A GeoJSON object with the type "Feature" is a feature object.
 A GeoJSON object with the type "FeatureCollection" is a feature
 collection object. A feature collection object MUST have a member with
 the name "features". The value of "features" is an array. Each element
-of the array is a feature object as defined above. It is possible for
-this array to be empty.
+of the array MUST be a feature object as defined above. It is possible
+for this array to be empty. The values in the array are ordered, meaning
+that GeoJSON implementations MUST maintain order when processing them.
```

Comment:
I'm happy with "The values in the array are ordered." period. 

What if I want to build a GeoJSON implementation that sorts features in a feature collection. If someone comes along in a few years, doesn't understand the history, and makes a conformance test for GeoJSON processors, then my sorter fails.

This is why I don't like talking about processing in this draft.